pom.xml

```diff
@@ -55,12 +55,6 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-test</artifactId>
             <scope>test</scope>
-            <exclusions>
```

Comment:
Please, don't remove it - we don't need vintage engine for our tests. 
Let's keep using JUnit5.